04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -241,54 +257,51 @@ minValue(RGB_band1_HARV)
 maxValue(RGB_band1_HARV)
 ```
 
-In this raster image, we see values between 0 and 255 are used. These values 
+In this raster image, we see values between 0 and 255. These values 
 represent degrees of brightness associated with the band color being viewed. In 
-the case of a RGB image (red, green and blue), band 1 is the red band. Thus when 
-you plot that band, larger numbers (towards 255) represent pixels with more red 
+the case of a RGB image (red, green and blue), band 1 is the red band. Thus when
+we plot that band, larger numbers (towards 255) represent pixels with more red 
 in them. Smaller numbers (towards 0) represent pixels with less red in them. To 
 plot an RGB image, we mix red + green + blue values into one single color.
 
 ##Import A Specific Band
 We can use the raster function to import specific bands in our raster object by
 specifying which band we want with `band=N` where N is the number of the band
-that you want to work with. If we want to import the green band we would use 
+that we want to work with. If we want to import the green band we would use 
 `band=2`.
 
 ```{r read-specific-band }
-# Can specify which band you want to read in
+# Can specify which band we want to read in
 RGB_band2_HARV <- raster("NEON_RemoteSensing/HARV/HARV_RGB_Ortho.tif", 
                     band = 2)
 
 #plot band 2
 plot(RGB_band2_HARV,
      col=grayscale_colors,   #we already created this palatte & can use it again
-     main="NEON RGB Imagery - Band 2\nHarvard Forest")
+     main="NEON RGB Imagery - Band 2- Green\nHarvard Forest")
 
 #view attributes of band 2 
 RGB_band2_HARV
 
 ```
 
-Notice that band 2 is band 2 of 3 bands.
-`band: 2  (of  3  bands)`
-
+Notice that band 2 is band 2 of 3 bands `band: 2  (of  3  bands)`.  
 
-> #Challenge: Making sense of single band images
-> If you compare the plots of band 1 (red) and band 2 (green) you can see subtle 
-> differences in the image. The forest appears paler in band 2 (green). Why?  
 
-Next, we will explore working with all 3 bands - using a `RasterStack`. 
+#Challenge: Making Sense of Single Band Images
+If you compare the plots of band 1 (red) and band 2 (green) you can see subtle 
+differences in the image. The forest appears paler in band 2 (green). Why?  
 
-```{r challenge1-answer, echo=FALSE }
+```{r challenge1-answer, eval=FALSE, echo=FALSE }
 
 #We'd expect a *brighter* value for the forest in band 2 (green) than in 
 #band 1 (red) because most trees appear green. 
```

Comment:
because the leaves on trees of most often "green" - that is the visible color that is reflected most strongly off of the leaves.